platform/measurement_utils.cpp

```diff
@@ -39,19 +39,20 @@ string FormatDistanceImpl(Units units, double m, string const & low, string cons
   case Units::Metric: highF = 1000.0; lowF = 1.0; break;
   }
 
-  double const lowV = m / lowF;
-  if (lowV < 1.0)
-    return string("0 ") + low;
+  double lowV = round(m / lowF);
+  // Round distances over 100 units to 10 units, e.g. 112 -> 110, 998 -> 1000
+  lowV = lowV <= 100.0 ? lowV : round(lowV / 10) * 10;
```

Comment:
Maybe rounding to 5 would be better, or no rounding at all. I really like the precise distance when approaching complex crossings slowly. What issue is solved by rounding? "Draw it as everyone"?

Reply:
That's just the way it was before my change.

I think the rounding reduces the number of meaningful digits so that people don't spend their attention on insignificant stuff (and changes of it).

E.g. we can display 3824m instead of 3.8km too, but 3.8km is just easier & faster to comprehend. Also its likely to change constantly by a few meters due to gps inaccuracy - it'll create a visual noise.

For your "precise distance when approaching complex crossings" use case - does it really matter you're 140m before the intersection or 136m? Its just a question of how precise it should be and for which distance (e.g. 300-500m before instead of current 100m?).

Reply:
There are complex crossings where +-30 meters distance really matters. I'm not speaking about 3833 meters accuracy (3.8 is enough), it's about distances < 1000 or even < 500m.

Reply:
Well current rounding accuracy for 100-1000m range is 10m, so "+-30 meters distance really matters" fits in here, no? Or did you mean +-3 meters?

Anyway, what is your preferred solution? Round to 5m starting from 100m or round to 10m starting from 500m or something else? I'm happy to change it this way. The only option I don't like is "no rounding at all".